ann_cnn_training/training.py

```diff
@@ -86,17 +86,27 @@ def write_log(*args):
     f"Training the {stencil}x{stencil} ANN-CNNs, {domain} horizontal and {vertical} vertical model with features {features} with min-max learning rates {lr_min} to {lr_max} for a CyclicLR, and dropout={dropout}.\n"
 )
 
+idir = sys.argv[5]
+odir = sys.argv[6]
```

Comment:
I would consider using a library like [`argparse`](https://docs.python.org/3/library/argparse.html) or [`click`](https://click.palletsprojects.com/en/stable/) to handle command-line options. Generally speaking they offer more protection. What is idir and odir don't exist? or what if you pass an integer by mistake etc.?